CONTRIBUTING.md

```diff
@@ -120,7 +120,11 @@ cd build
 cmake -Duse_default_uuid=ON ..
 cmake --build .
 ```
-
+If you want to run tests also, generate build files using below command and then build.
+```sh
+cmake -Duse_default_uuid=ON -DBUILD_TESTING=ON -DBUILD_CURL_TRANSPORT=ON ..
```

Comment:
oh, this file and readme are totally out of date.
Thank you for looking into it.
this option `use_default_uuid` is not required anymore.